dummy/src/main/java/org/ldbcouncil/finbench/impls/dummy/DummyDb.java

```diff
@@ -239,6 +272,7 @@ public static class SimpleRead8Handler implements OperationHandler<SimpleRead8,
         public void executeOperation(SimpleRead8 sr8, DummyDbConnectionState dummyDbConnectionState,
                                      ResultReporter resultReporter) throws DbException {
             DummyDb.logger.info(sr8.toString());
+            resultReporter.report(0, Collections.EMPTY_LIST, sr8);
```

Comment:
why we need report empty here?

Reply:
It will be throw exception if queries don't set results.